asf_lifetime_cost_model/pipeline/lifetime_cost_calculator.py

```diff
@@ -0,0 +1,354 @@
+        # replace the wholesale costs with the projection for that year
+
+        # Gas costs
+        gas_unit_cost_time_series, gas_standing_charge_time_series = create_energy_cost_time_series(
+            tariff_time_series=gas_tariff_time_series,
+            fuel_type="gas",
+            wholesale_prices_series=wholesale_prices_series_df,
+        )
+
+        # Electricity costs
+        electricity_unit_cost_time_series, electricity_standing_charge_time_series = create_energy_cost_time_series(
+            tariff_time_series=electricity_tariff_time_series,
+            fuel_type="electricity",
+            wholesale_prices_series=wholesale_prices_series_df,
+        )
+
+        if include_vat:
+            for cost_time_series in [
+                gas_unit_cost_time_series,
+                gas_standing_charge_time_series,
+                electricity_unit_cost_time_series,
+                electricity_standing_charge_time_series,
+            ]:
+                for year in cost_time_series:
+                    cost_time_series[year] *= 1.05
+
+        ### Compute total running cost in each year of operation ###
+        running_costs_time_series = pd.DataFrame()
+        for year in years_of_operation:
+            if fuel_type == "gas":
+                running_costs_time_series[year] = gas_unit_cost_time_series[year] * (
+                    heat_demand_data["energy_demand"] / 1_000
+                )  # convert energy_demand in kWh/year to MWh/year
+
+                if include_standing_charge:
+                    running_costs_time_series[year] += gas_standing_charge_time_series[year]
+
+            else:  # fuel_type is electricity
+                running_costs_time_series[year] = electricity_unit_cost_time_series[year] * (
+                    heat_demand_data["energy_demand"] / 1_000
+                )
+
+                if include_standing_charge:
+                    running_costs_time_series[year] += electricity_standing_charge_time_series[year]
+
+        return running_costs_time_series
+
+    def compute_total_lifetime_costs(
+        self, installation_costs: pd.DataFrame, maintenance_costs: pd.DataFrame, running_costs: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Computes the lifetime cost of a heating system over its lifetime.
+
+        Args:
+            installation_costs (pd.DataFrame): DataFrame containing installation costs for different property archetypes.
+            maintenance_costs (pd.DataFrame): DataFrame containing total maintenance costs for different property archetypes.
+            running_costs (pd.DataFrame): DataFrame containing total running costs for different property archetypes in each year of operation.
+
+        Returns:
+            pd.DataFrame: A DataFrame with property archetypes as index and total lifetime costs as column.
+        """
+        total_lifetime_costs = pd.DataFrame()
+        total_lifetime_costs["lifetime_installations"] = installation_costs
```

Comment:
```suggestion
        total_lifetime_costs["lifetime_installation_costs"] = installation_costs.sum(axis=1)
```

1. Fix typo in dataframe column name
2. If we decide to implement my other suggestion on separating out loan amount, then this summation is needed